internal/api/openapi.json

```diff
@@ -140,6 +140,9 @@
                         "properties": {
                             "account": {
                                 "type": "string"
+                            },
+                            "org_id": {
+                                "type": "string"
```

Comment:
Would be nice to document the new field by giving it a description. The canonical description for org_id can be found in https://docs.google.com/document/d/19dEXx087osJ8cNmbY5LvDEZKGDytjY9RifJBIP5mME4/edit